maxim/logger/gemini/client.py

```
@@ -72,18 +72,18 @@ def send_message(
+                "model": self._model,
+                "provider": "google",
+                "name": generation_name,
+                "model_parameters": GeminiUtils.get_model_params(config),
+                "messages": messages,
+            }
+            generation = self._logger.trace_add_generation(self._trace_id, gen_config)
             # Attaching history as metadata
             if self._curated_history is not None:
-                generation.add_metadata({"history": self._curated_history})
+                generation.add_metadata({"history": self._comprehensive_history})
         except Exception as e:
```

Ruff (0.11.9)

Do not catch blind exception: `Exception`

(BLE001)